sorc/ncep_post.fd/WRFPOST.f

```diff
@@ -273,6 +275,7 @@ PROGRAM WRFPOST
         fileNameFlat='postxconfig-NT.txt'
         read(5,nampgb,iostat=iret,end=119)
  119    continue
+       if (me==0) print*,'in itag, write_ifi_debug_files=', write_ifi_debug_files
```

Comment:
@SamuelTrahanNOAA suggest setting a default value for write_ifi_debug_file=.false.

Reply:
It has a default of false. The variable comes from IFI.F
```f90
use upp_ifi_mod, only: write_ifi_debug_files
```

and it is initialized to false in both the stub and actual IFI wrappers:

```f90
! IFI stubs

module upp_ifi_mod
...
logical :: write_ifi_debug_files = .false.
...
#else
! Actual IFI code.
module upp_ifi_mod
logical, public :: write_ifi_debug_files = .false.
```

Reply:
great.  will mark it as resolved